augur/util_support/auspice_config.py

```diff
@@ -182,6 +185,7 @@ def remove_unused_metadata_columns(columns: list) -> list:
     {"old_name": "geo", "new_name": "geo_resolutions", "modify": lambda values: [{"key": v} for v in values]},
     {"old_name": "color_options", "new_name": "colorings", "modify": _parse_color_options},
     {"old_name": "colorings", "new_name": "colorings", "modify": _rename_deprecated_authors_coloring},
+    {"old_name": "filters", "new_name": "filters", "modify": _rename_deprecated_filters},
```

Comment:
tangential / not blocking. The "filters" list is used to populate the dropdowns in the page footer, and continuous variables (e.g. `num_date`) are filtered out via

https://github.com/nextstrain/auspice/blob/e40a221785d24c19d937af83443cd8d877275264/src/actions/recomputeReduxState.js#L646-L647

which drops keys of continuous colourings via

https://github.com/nextstrain/auspice/blob/e40a221785d24c19d937af83443cd8d877275264/src/util/treeCountingHelpers.ts#L56-L59

Reply:
Ah, good to know. I was wondering why we haven't gotten any bug reports regarding the `numdate`/`num_date` mismatch.